client/keys/show.go

```diff
@@ -18,30 +22,67 @@ const (
 	FlagPublicKey = "pubkey"
 	// FlagBechPrefix defines a desired Bech32 prefix encoding for a key.
 	FlagBechPrefix = "bech"
+
+	flagMultiSigThreshold = "multisig-threshold"
 )
 
+var _ keys.Info = (keys.Info)(nil)
```

Comment:
Why is this here? Aren't `keys.Info` the same thing?

Reply:
I made `multiSigKey` meet `keys.Info` to avoid writing specific functions for `github.com/tendermint/tendermint/crypto/multisig` types

Reply:
Shouldn't it be ` var _ keys.Info = (*multiSigKey)(nil)`?

Reply:
Jeez, yes of course

Reply:
Fixed